code/src/components/ProgressBar.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+const ProgressBar = (props) => {
+  const { completed, totalQuestions } = props;
+
+  const containerStyles = {
+    height: 20,
+    width: '90%',
+    backgroundColor: '#e0e0de',
+    borderRadius: 50,
+    marginTop: 30,
+  };
+
+  const fillerStyles = {
+    height: '100%',
+    width: `${(completed / totalQuestions) * 100}%`,
+    backgroundColor: '#00695c',
+    borderRadius: 'inherit',
+    textAlign: 'right',
+  };
+
+  const labelStyles = {
+    paddingRight: 5,
+    color: 'white',
+    letterSpacing: -1,
+    fontWeight: 'bold',
+  };
```

Comment:
Yeah, sometimes it's easier to style within the same file instead of switching back and forward between the js and css file. As long as the code is not super long in the js 😄